include/pacmod3/pacmod3_node.hpp

```diff
@@ -80,6 +80,26 @@ class PACMod3Node final
   LNI::CallbackReturn on_error(const lc::State & state) override;
 
 private:
+  void initializeBrakeMotorRptApi();
```

Comment:
~PACMod3Node();
the destructor should be virtual.

Reply:
See pacmod3_node.cpp, there is an explicit destructor since a thread needs to be joined.